src/lib.rs

```diff
@@ -16,15 +16,17 @@ mod benchmarking;
 pub mod weights;
 pub use weights::*;
 
+pub const INCENTIVE_SHARE: u8 = 10; // TODO configure value
```

Comment:
perhaps a more explicit name for this variable could be something like "janitor incentive", to make it obvious what it is meant for

Reply:
also (as the `TODO` comment states) this ideally should be a `Config` constant type

Reply:
Yes, agree, can be moved to `Config`

Reply:
Oh, true, they can call the `clear_expired_match` from a different account. Indeed...

Reply:
But then, so what? The `winner` gets 90% anyway, and, it's an incentive for them not to ignore the game and clear up the storage on time! :D

Reply:
that's true

the thing that bothers me is that the bad loser still has an incentive to refuse to play, and that kind of changes the dynamics of the game a little bit

a rational player on this pallet wouldn't be a "canonical chess player" anymore... their strategy wouldn't be 100% guided by the rules of chess if they want to find ways to cheat

but I totally agree that's not a big deal! I guess we can merge this solution as it is and find a way to improve this later